### ISSUE TITLE
DJ Hero 2 - PS3 Turntable Inputs Not Working

### ISSUE THREAD

**Issue by archonpy** [RPCS3/rpcs3#13788]
### Quick summary

Followed the instructions to configure PS3 DJ Hero turntable using zadig to update driver to winusb. Ensured emulated turntable is set to Null. DJ Hero 1 works flawlessly , but DJ Hero 2 shows turntable detected but when the game boots up none of the turntable inputs are working.

### Details

I was going to submit this as a regression ticket based on the discord channel and prior tickets indicating this worked at one point. However, i tried several prior builds to include going back to 2020 when this was first supported. I also tried on multiple Windows 10 machines and experienced the same behavior where DJ Hero 1 works perfectly while DJ Hero 2 does not respond to any of the ps3 turntable's input.  Tested on DJ Hero 2 BLES00896, DJ Hero BLES00602. Also tried the US version of DJ Hero 2 with same behavior

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11360419/RPCS3.log.gz)
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/11360420/RPCS3.log)

### System configuration

RPCS3 v0.0.27-14944-7bec3b30 Alpha | master | FW 4.90 | Windows 10 21H2
AMD Ryzen 7 3700X | 16 Threads | 15.93 GiB RAM | AVX+ | FMA3
GPU: NVIDIA GeForce RTX 3060 Ti (516.94)

**Comment by Crafty-The-Fox** [comment]
Pretty sure this is a duplicate of [](https://github.com/RPCS3/rpcs3/issues/13788)

**Comment by EverdayPlane** [comment]
Tested in the last build, the problem persists, maybe one day they will be able to fix this, the controls don't recognize it, even after setting the correct settings I spent more than 3 hours trying to fix the control. But apparently the problem is in the emulator itself. I hope one day they can fix this, Tested with PS4, Xbox360 controls, the fake function. Dj Hero Turntable emulation is not working.

<a href="https://ibb.co/FJ9Rnvv"><img src="https://i.ibb.co/zb1qNcc/DJ-hero2.png" alt="DJ-hero2" border="0"></a>
